### PR TITLE
Continue 5821

### PR DESCRIPTION
**Pull request by stuartarchibald**
Continues PR #5821 addressing feedback from review.

Closes #5821

### CODE REVIEW

**Comment by stuartarchibald** [comment]
@esc please could you add this to your review queue, thanks.

File: setup.py [hunk 1]
```diff
@@ -205,33 +201,42 @@ def check_file_at_path(path2file):
...
-                os.path.join(tbb_root, 'lib', 'intel64', 'gcc4.4'),
-                # for MacOS
-                os.path.join(tbb_root, 'lib'),
-                # for Windows
-                os.path.join(tbb_root, 'lib', 'intel64', 'vc_mt'),
-            ],
-        )
-        ext_np_ufunc_backends.append(ext_np_ufunc_tbb_backend)
+    # Disable tbb if forced by user with NUMBA_DISABLE_TBB=1
+    if os.getenv("NUMBA_DISABLE_TBB"):
```
Comment (esc) [inline setup.py:205]: The problem with this is that a Falsy value like `0` will become a string with length 1 --- which is a truthy. I'm not sure what the best solution here is, maybe it's best to state in the documentation that the variable must be set to a non-empty string? I think this is the most  accurate description.

File: setup.py [hunk 2]
```diff
@@ -205,33 +201,42 @@ def check_file_at_path(path2file):
...
+                ],
+            )
+            ext_np_ufunc_backends.append(ext_np_ufunc_tbb_backend)
+        else:
+            print("TBB not found")
 
-    # Disable OpenMP if forced by user with NUMBA_NO_OPENMP=1
-    if os.getenv('NUMBA_NO_OPENMP'):
+    # Disable OpenMP if forced by user with NUMBA_DISABLE_OPENMP=1
+    if os.getenv('NUMBA_DISABLE_OPENMP'):
```
Comment (esc) [inline setup.py:239]: As above

File: setup.py (hunk 1 above)
Comment (stuartarchibald) [reply setup.py:205]: Good point, done in a02bdea

**Pull request merged**